components/layout_2020/display_list/stacking_context.rs

```diff
@@ -750,37 +733,39 @@ impl AbsoluteOrFixedPositionedFragment {
         containing_block_info: &ContainingBlockInfo,
         stacking_context: &mut StackingContext,
     ) {
-        let mut build_for_containing_block = |containing_block: &ContainingBlock| {
-            let hoisted_child = match containing_block.hoisted_children.get(&self.0) {
-                Some(hoisted_child) => hoisted_child,
-                None => return false,
-            };
-
-            builder.clipping_and_scrolling_scope(|builder| {
-                let mut new_containing_block_info = containing_block_info.clone();
-                new_containing_block_info.rect = containing_block.rect;
-                builder.current_space_and_clip = containing_block.space_and_clip;
-                hoisted_child.borrow().build_stacking_context_tree(
-                    hoisted_child,
-                    builder,
-                    &new_containing_block_info,
-                    stacking_context,
-                    StackingContextBuildMode::IncludeHoisted,
-                );
-            });
+        let hoisted_fragment = self.hoisted_fragment.borrow();
+        let fragment_ref = match hoisted_fragment.as_ref() {
+            Some(fragment_ref) => fragment_ref,
+            None => {
+                warn!("Found hoisted box with missing fragment.");
```

Comment:
Yeah, this does indicate a bug and I'm actually looking at this now, because changing this to `unreachable!` seems to have uncovered a few errors. In this change I didn't modify it because this code was already a `warn!` for this situation and I wanted to keep this change separate from the fix.

Reply:
Oh I didn’t realize this was already a `warn`. Let’s land this as-is, and please look into that issue next.